tests/DataBuilderTest.php

```diff
@@ -892,7 +889,11 @@ public function testFramesOrder()
             $frames[count($frames)-1]->getFilename()
         );
         // 889 is the line number where the comment "// A" is found
-        $this->assertEquals(889, $frames[count($frames)-1]->getLineno());
+        $this->assertEquals(
+            886,
+            $frames[count($frames)-1]->getLineno(),
+            "Possible false negative: did this file change? Check the line number for line with '// A' comment"
+        );
```

Comment:
This test uses the test file itself as the input source for frames. Consequently, when the test file changes, the test may fail. The change here adds a comment, which will be output to the test runner, that the test failure may be false and should be cross-checked against the line number. Before, this same information was in the file itself as only a comment: now we also bubble this up to the test failure itself for greater visibility.